ChangeLog

```diff
@@ -36,6 +36,8 @@ Changes
    * Clarify ECDSA documentation and improve the sample code to avoid
      misunderstandings and potentially dangerous use of the API. Pointed out
      by Jean-Philippe Aumasson.
+   * Tweaked apidoc_full.sh to include functions with alternative
```

Comment:
What tense are we supposed to use in the ChangeLog?

We should use a consistent tense.

Reply:
Judging from the surrounding ChangeLog entries, this should be `Tweak` for uniformity.

Reply:
I've changed the tense and tried to improve the comment.